mbuild/conversion.py

```diff
@@ -200,44 +220,82 @@ def load_smiles(smiles_or_filename,
     infer_hierarchy : bool, optional, default=True
     ignore_box_warn : bool, optional, default=False
         If True, ignore warning if no box is present.
+    backend : str, optional, default='rdkit'
+        The smiles loading backend, either 'rdkit' or 'pybel'
+    smiles_seed : int, optional, default=0
+        The random number seed to control the position of particles generated
+        from the smiles loader. -1 means no seed is set.
+
 
     Returns
     -------
     compound : mb.Compound
 
     """
-    # Will try to support list of smiles strings in the future
-    pybel = import_('pybel')
-
     # Initialize an mb.Compound if none is provided
     if not compound:
         compound = mb.Compound()
 
-    # First we try treating filename_or_object as a SMILES string
-    try:
-        mymol = pybel.readstring("smi", smiles_or_filename)
-    # Now we treat it as a filename
-    except(OSError, IOError):
-        # For now, we only support reading in a single smiles molecule,
-        # but pybel returns a generator, so we get the first molecule
-        # and warn the user if there is more
-
-        mymol_generator = pybel.readfile("smi", smiles_or_filename)
-        mymol_list = list(mymol_generator)
-        if len(mymol_list) == 1:
-            mymol = mymol_list[0]
+    test_path = Path(smiles_or_filename)
+
```

Comment:
```suggestion

if backend is None:
    backend = "rdkit"
    
```
The backend default from `mb.load` is `None`. Since `load_smiles` needs a backend, I think we should automatically handle that here.